contracts/makefile

```diff
@@ -236,8 +236,8 @@ interact-river :;
 # 				 		Base
 # ===========================
 deploy-base :;
-	@$(MAKE) deploy-any-ledger rpc=base hd_path=${HD_PATH} sender=${SENDER_ADDRESS}
-	@$(MAKE) resume-any-ledger rpc=base hd_path=${HD_PATH} sender=${SENDER_ADDRESS} verifier=${BASESCAN_URL} etherscan=${BASESCAN_API_KEY}
+	@$(MAKE) deploy-any-ledger rpc=base hd_path="${HD_PATH}" sender=${SENDER_ADDRESS}
```

Comment:
Was this quoting needed?

Reply:
Yea even if you escape single quotes in hd path in your env_ it breaks running deploy looking for a close '